docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>plot the .tif file using 'matplotlib'</p><p> </p><p>&gt;&gt; Plot the .tif file using 'matplotlib'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>